client-lite/src/download/download.cpp

```diff
@@ -1,6 +1,7 @@
 #include "do_common.h"
 #include "download.h"
 
+#include <cmath>
```

Comment:
is this unused?

Reply:
It is used for std::round on line 455. I think it was being brought in by one of cpprest's headers earlier.